services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy

```diff
@@ -164,6 +164,18 @@ class LinkPropertiesSpec extends BaseSpecification {
         islUtils.getIslInfo(updatedLinks, isl).get().maxBandwidth == maxBandwidthValue.toInteger()
         islUtils.getIslInfo(updatedLinks, isl.reversed).get().maxBandwidth == maxBandwidthValue.toInteger()
 
+        when: "Update cost on forward direction of the link props"
+        def newCostValue = "345"
+        def newLinkPropsInfo = northbound.getAllLinkProps().first().tap { it.props.cost = newCostValue }
+        northbound.updateLinkProps([newLinkPropsInfo])
+
+        then: "Forward and reverse directions of the link props are really updated"
+        northbound.getAllLinkProps().each {
+            assert it.props.cost == newCostValue
+        }
+        database.getIslCost(isl) == newCostValue.toInteger()
```

Comment:
This check should be in a separate `and` block with description `Cost on forward and reverse ISLs is really updated`

Reply:
new block was added